aopy/precondition/eye.py

```diff
@@ -193,3 +195,324 @@ def detect_saccades(eye_pos, samplerate, thr=None, num_sd=1.5, intersaccade_min=
+    )
+
+    events=[]
+
+    for eye_trajectory in eye_trajectories:
+        ev=classify_event_in_eye_trajectory(eye_trajectory, clf_params, preproc_params, screen_half_height, viewing_dist, samplerate)
+        events.append(ev)
+    
+    return events
+
+def get_saccade_trials (preproc_dir, subject, te_id, date, start_events, end_events, clf_params, preproc_params, screen_half_height, viewing_dist, samplerate=1000):
+    """
+        Extracts saccade event times and positions from preprocessed eye-tracking data.
+
+        Args:
+            preproc_dir (str): Base directory where file lives
+            subject (str): Subject name.
+            te_id (int): Block number of task entry object.
+            date (str): Date of the recording.
+            start_events (list): List of numeric codes representing the start of a trial.
+            end_events (list): List of numeric codes representing the end of a trial.
+            clf_params (dict): Dictionary of classifier parameters to be passed to the REMoDNaV EyegazeClassifier.
+            preproc_params (dict): Dictionary of preprocessing parameters to be passed to the classifier's preproc method.
+            screen_half_height (float): Half the height of the screen in cm.
+            viewing_dist (float): The viewing distance in cm.
+            samplerate (int, optional): The sampling rate of the eye tracker in Hz. Default is 1000.
+
+        Returns:
+            start_end_times (start, end): 
+                List of arrays containing start and end times of saccades for each trial in seconds.
+            start_pos (start_x, start_y):
+                List of arrays containing start positions of saccades for each trial.
+            end_pos : (end_x, end_y):
+                List of arrays containing end positions of saccades for each trial.
+        """
+    times, start, end=get_eye_event_trials(preproc_dir, subject, te_id, date, start_events, end_events, 'SACC', clf_params, preproc_params, screen_half_height, viewing_dist, samplerate)
+    return (times, start, end)
+
+def get_pursuit_trials (preproc_dir, subject, te_id, date, start_events, end_events, clf_params, preproc_params, screen_half_height, viewing_dist, samplerate=1000):
+    """
+        Extracts pursuit event times and positions from preprocessed eye-tracking data.
+
+        Args:
+            preproc_dir (str): Base directory where file lives
+            subject (str): Subject name.
+            te_id (int): Block number of task entry object.
+            date (str): Date of the recording.
+            start_events (list): List of numeric codes representing the start of a trial.
+            end_events (list): List of numeric codes representing the end of a trial.
+            clf_params (dict): Dictionary of classifier parameters to be passed to the REMoDNaV EyegazeClassifier.
+            preproc_params (dict): Dictionary of preprocessing parameters to be passed to the classifier's preproc method.
```

Comment:
whereever you have these arguments it would be helpful to mention the `get_default_parameters` function. if you format it as 
```
:func:`~aopy.precondition.eye.get_default_parameters`
```
it should appear as a link